R/server.R

```diff
@@ -193,16 +192,27 @@ app_server <- function(input, output, session) {
   output$usage_plot <- renderPlotly({
     withProgress(message = "Plotting data ...", {
       unique_codes <- length(unique(filtered_data()$code))
-      
+
+      # As a workaround we are adding a plot with text only if the
+      # search criteria match no data. At some point in the future we
+      # should reconsider if this is the best approach.
+       text if there are no codes
```

Comment:
```suggestion
       # text if there are no codes
```

oh sorry, this should still be a comment